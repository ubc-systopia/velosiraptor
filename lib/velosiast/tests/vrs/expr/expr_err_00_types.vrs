true + 1;
1 && 2;
1 < true;
1 + true;
a && true;