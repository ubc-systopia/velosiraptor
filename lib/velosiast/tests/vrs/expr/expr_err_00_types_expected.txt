error: Unsupported type combination in binary operation
     --> tests/vrs/expr/expr_err_00_types.vrs:1:1
      |
    1 |  true + 1;
      |  ^^^^^^^^ No implementation for `bool + int`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

error: Unsupported type combination in binary operation
     --> tests/vrs/expr/expr_err_00_types.vrs:2:1
      |
    2 |  1 && 2;
      |  ^^^^^^ No implementation for `int && int`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

error: Unsupported type combination in binary operation
     --> tests/vrs/expr/expr_err_00_types.vrs:3:1
      |
    3 |  1 < true;
      |  ^^^^^^^^ No implementation for `int < bool`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

error: Unsupported type combination in binary operation
     --> tests/vrs/expr/expr_err_00_types.vrs:4:1
      |
    4 |  1 + true;
      |  ^^^^^^^^ No implementation for `int + bool`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

error: Unsupported type combination in binary operation
     --> tests/vrs/expr/expr_err_00_types.vrs:5:1
      |
    5 |  a && true;
      |  ^^^^^^^^^ No implementation for `int && bool`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
